### PR TITLE
[RHPAM-2591] Task identifers are recorded on the DB with negative numbers

### PR DESCRIPTION
**Pull request by elguardian**
changed oracle scripts to reflect properly the increment by = allocation size in entities.

### CODE REVIEW

File: jbpm-db-scripts/src/main/resources/db/ddl-scripts/oracle/oracle-jbpm-schema.sql [hunk 1]
```diff
@@ -708,73 +708,40 @@
...
+	create sequence QUERY_DEF_ID_SEQ start with 1 increment by 1;
+	create sequence REASSIGNMENT_ID_SEQ start with 1 increment by 1;
+	create sequence REQUEST_INFO_ID_SEQ start with 1 increment by 50;
+	create sequence SESSIONINFO_ID_SEQ start with 1 increment by 50;
+	create sequence TASK_DEF_ID_SEQ start with 1 increment by 50;
+	create sequence TASK_EVENT_ID_SEQ start with 1 increment by 50;
+	create sequence TASK_ID_SEQ start with 1 increment by 1;
+	create sequence TASK_VAR_ID_SEQ start with 1 increment by 1;
+	create sequence VAR_INST_LOG_ID_SEQ start with 1 increment by 1;
+	create sequence WORKITEMINFO_ID_SEQ start with 1 increment by 50;
```
Comment (afalhambra) [inline jbpm-db-scripts/src/main/resources/db/ddl-scripts/oracle/oracle-jbpm-schema.sql:743]: Why this seq sets `increment by` to 50? It's not referenced by any entity in the code.

File: jbpm-db-scripts/src/main/resources/db/ddl-scripts/oracle/oracle-jbpm-schema.sql [hunk 2]
```diff
@@ -708,73 +708,40 @@
...
+	create sequence EXEC_ERROR_INFO_ID_SEQ start with 1 increment by 1;
+	create sequence I18NTEXT_ID_SEQ start with 1 increment by 1;
+	create sequence NODE_INST_LOG_ID_SEQ start with 1 increment by 1;
+	create sequence NOTIFICATION_ID_SEQ start with 1 increment by 1;
+	create sequence PROC_INST_LOG_ID_SEQ start with 1 increment by 1;
+	create sequence PROCESS_INSTANCE_INFO_ID_SEQ start with 1 increment by 50;
+	create sequence QUERY_DEF_ID_SEQ start with 1 increment by 1;
+	create sequence REASSIGNMENT_ID_SEQ start with 1 increment by 1;
+	create sequence REQUEST_INFO_ID_SEQ start with 1 increment by 50;
+	create sequence SESSIONINFO_ID_SEQ start with 1 increment by 50;
```
Comment (afalhambra) [inline jbpm-db-scripts/src/main/resources/db/ddl-scripts/oracle/oracle-jbpm-schema.sql:737]: This sequence is only used by a test java class, so it won't really be used in a production environment.
Maybe we can keep it for consistency. Wdyt?

File: jbpm-db-scripts/src/main/resources/db/ddl-scripts/oracle/oracle-jbpm-schema.sql (hunk 1 above)
Comment (elguardian) [reply jbpm-db-scripts/src/main/resources/db/ddl-scripts/oracle/oracle-jbpm-schema.sql:743]: allocation size in jpa entity = INCREMENT BY

File: jbpm-db-scripts/src/main/resources/db/ddl-scripts/oracle/oracle-jbpm-schema.sql (hunk 2 above)
Comment (elguardian) [reply jbpm-db-scripts/src/main/resources/db/ddl-scripts/oracle/oracle-jbpm-schema.sql:737]: Hi, the old DDL has this sequence. So it matches the old thing.

**Pull request merged**